src/js/views/globals/search/patient-search.scss

```diff
@@ -69,3 +71,41 @@
   padding: 16px 0;
   text-align: center;
 }
+
+.patient-search__tip {
+  border-bottom: 1px solid $black-90;
+  padding-bottom: 16px;
+  padding-left: 60px;
+  text-align: left;
+}
+
+.patient-search__search-by {
+  padding: 16px 0;
+  padding-left: 60px;
+  text-align: left;
+}
+
+.patient-search__search-by-title {
+  font-size: 12px;
+  font-weight: 600;
+}
+
+.patient-search__search-by-label {
```

Comment:
keep this style in mind.. if we end up using it somewhere else we should genericize it once we know how it can be described generally @nmajor25